### ISSUE TITLE
.Net Core error

### ISSUE THREAD

**Issue by lbargaoanu** [agileobjects/ReadableExpressions#69]
An unhandled exception of type 'System.TypeInitializationException' was thrown by the custom visualizer component in the process being debugged. The type initializer for 'AgileObjects.ReadableExpressions.Visualizers.Core.Configuration.VisualizerDialogSettings' threw an exception.
```
   at AgileObjects.ReadableExpressions.Visualizers.Core.Configuration.VisualizerDialogSettings.GetInstance()
   at AgileObjects.ReadableExpressions.Visualizers.Core.ExpressionVisualizerObjectSource.GetDialogSettings()
   at AgileObjects.ReadableExpressions.Visualizers.Core.ExpressionVisualizerObjectSource.<>c.<Translate>b__4_0(TranslationSettings settings)
   at AgileObjects.ReadableExpressions.ExpressionExtensions.GetTranslationSettings(Func`2 configuration)
   at AgileObjects.ReadableExpressions.ExpressionExtensions.ToReadableString(Expression expression, Func`2 configuration)
   at AgileObjects.ReadableExpressions.Visualizers.Core.ExpressionVisualizerObjectSource.Translate(Expression expression)
   at AgileObjects.ReadableExpressions.Visualizers.Core.ExpressionVisualizerObjectSource.GetTranslationFor(Object target)
   at AgileObjects.ReadableExpressions.Visualizers.Core.ExpressionVisualizerObjectSource.GetData(Object target, Stream outgoingData, Action`2 serializer)
   at AgileObjects.ReadableExpressions.Visualizers.Vs16ExpressionVisualizerObjectSource.GetData(Object target, Stream outgoingData)
   at Microsoft.VisualStudio.DebuggerVisualizers.DebuggeeSide.Impl.ClrCustomVisualizerDebuggeeHost.GetData(Object visualizedObject)```

**Comment by pheber** [comment]
I'm having the same issue.

**Comment by SteveWilkes** [comment]
Thanks for raising this - I'm looking into it.

**Comment by pollau** [comment]
I have the same here. It happens when trying to open the visualizer

**Comment by SteveWilkes** [comment]
Thanks again for the feedback, everyone - this should be fixed as of v3.2.2 of the Visualizers, which is now in [the VS Marketplace](https://marketplace.visualstudio.com/items?itemName=vs-publisher-1232914.ReadableExpressionsVisualizers).
